src/components/Card.vue

```diff
@@ -25,46 +25,46 @@ export default {
 </script>
 
 <style scoped>
-  .card {
-    background-color: #131315;
-    color: #fff;
-    text-align: left;
-  }
-  .avatar {
-    width: 53px;
-    height: 53px;
-    position: absolute;
-    margin-top: -22px;
-    margin-left: -3px;
-  }
-  .title {
-    padding-top: 12px;
-    padding-left: 57px;
-    font-size: 18px;
-    font-family: Poppins;
-    font-weight: 700;
-    line-height: 25px;
-  }
-  .meta {
-    padding-top: 16px;
-    padding-left: 9px;
-    color: #717171;
-    font-family: Poppins;
-    font-size: 12px;
-    line-height: 17px;
-  }
-  .like {
-    cursor: pointer;
-    width: 24px;
-    height: 24px;
-    padding-bottom: 15px;
-    padding-left: 55px;
-  }
-  .comment {
-    cursor: pointer;
-    width: 24px;
-    height: 24px;
-    padding-bottom: 15px;
-    padding-left: 28px;
-  }
+.card {
+  background-color: #131315;
+  color: #fff;
+  text-align: left;
+}
+.avatar {
+  width: 53px;
```

Comment:
We are going to use rem, thats why we changed the html font size to 10px. Please adjust this.